pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "aws-connect"
+version = "0.1.0"
+description = ""
+authors = ["Szymon Bańczyk <sbanczyk02@interia.pl>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+boto3 = "^1.33.10"
+configparser = "^6.0.0"
```

Comment:
this is not the default `configparser` which is built-in python, but instead a custom library (not used in this project: https://pypi.org/project/configparser/ )